imagestreams/ruby-centos.json

```diff
@@ -68,6 +68,25 @@
           "type": "Local"
         }
       },
+      {
+        "name": "2.7",
+        "annotations": {
+          "openshift.io/display-name": "Ruby 2.7",
+          "openshift.io/provider-display-name": "Red Hat, Inc.",
+          "description": "Build and run Ruby applications on CentOS 7. For more information about using this builder image, including OpenShift considerations, see https://github.com/sclorg/s2i-ruby-container/tree/master/2.7/README.md.\n\nWARNING: By selecting this tag, your application will automatically update to use the latest version of Ruby available on OpenShift, including major version updates.",
+          "iconClass": "icon-ruby",
+          "tags": "builder,ruby",
```

Comment:
Add `,hidden` here, as in the RHEL/UBI equivalent.

Reply:
Good idea, done